File: OPENAPI-REST-API/swagger-client/java-vertx/src/main/java/io/swagger/server/api/model/InlineResponse2012OrderDetailsCjParams.java
package io.swagger.server.api.model;

import java.util.Objects;
import com.fasterxml.jackson.annotation.JsonProperty;
import com.fasterxml.jackson.annotation.JsonCreator;
import io.vertx.core.json.JsonObject;
import io.vertx.codegen.annotations.DataObject;
/**
 * InlineResponse2012OrderDetailsCjParams
 */




public class InlineResponse2012OrderDetailsCjParams   {
  private Integer containerTagId = null;

  private Integer CID = null;

  private String OID = null;

  private Integer TYPE = null;

  private String itEM1 = null;

  private Integer amT1 = null;

  private Integer qtY1 = null;

  private String CURRENCY = null;

  public InlineResponse2012OrderDetailsCjParams containerTagId(Integer containerTagId) {
    this.containerTagId = containerTagId;
    return this;
  }

  /**
   * Get containerTagId
   * @return containerTagId
   **/
    public Integer getContainerTagId() {
    return containerTagId;
  }

  public void setContainerTagId(Integer containerTagId) {
    this.containerTagId = containerTagId;
  }

  public InlineResponse2012OrderDetailsCjParams CID(Integer CID) {
    this.CID = CID;
    return this;
  }

  /**
   * Get CID
   * @return CID
   **/
    public Integer getCID() {
    return CID;
  }

  public void setCID(Integer CID) {
    this.CID = CID;
  }

  public InlineResponse2012OrderDetailsCjParams OID(String OID) {
    this.OID = OID;
    return this;
  }

  /**
   * Get OID
   * @return OID
   **/
    public String getOID() {
    return OID;
  }

  public void setOID(String OID) {
    this.OID = OID;
  }

  public InlineResponse2012OrderDetailsCjParams TYPE(Integer TYPE) {
    this.TYPE = TYPE;
    return this;
  }

  /**
   * Get TYPE
   * @return TYPE
   **/
    public Integer getTYPE() {
    return TYPE;
  }

  public void setTYPE(Integer TYPE) {
    this.TYPE = TYPE;
  }

  public InlineResponse2012OrderDetailsCjParams itEM1(String itEM1) {
    this.itEM1 = itEM1;
    return this;
  }

  /**
   * Get itEM1
   * @return itEM1
   **/
    public String getItEM1() {
    return itEM1;
  }

  public void setItEM1(String itEM1) {
    this.itEM1 = itEM1;
  }

  public InlineResponse2012OrderDetailsCjParams amT1(Integer amT1) {
    this.amT1 = amT1;
    return this;
  }

  /**
   * Get amT1
   * @return amT1
   **/
    public Integer getAmT1() {
    return amT1;
  }

  public void setAmT1(Integer amT1) {
    this.amT1 = amT1;
  }

  public InlineResponse2012OrderDetailsCjParams qtY1(Integer qtY1) {
    this.qtY1 = qtY1;
    return this;
  }

  /**
   * Get qtY1
   * @return qtY1
   **/
    public Integer getQtY1() {
    return qtY1;
  }

  public void setQtY1(Integer qtY1) {
    this.qtY1 = qtY1;
  }

  public InlineResponse2012OrderDetailsCjParams CURRENCY(String CURRENCY) {
    this.CURRENCY = CURRENCY;
    return this;
  }

  /**
   * Get CURRENCY
   * @return CURRENCY
   **/
    public String getCURRENCY() {
    return CURRENCY;
  }

  public void setCURRENCY(String CURRENCY) {
    this.CURRENCY = CURRENCY;
  }


  @Override
  public boolean equals(java.lang.Object o) {
    if (this == o) {
      return true;
    }
    if (o == null || getClass() != o.getClass()) {
      return false;
    }
    InlineResponse2012OrderDetailsCjParams inlineResponse2012OrderDetailsCjParams = (InlineResponse2012OrderDetailsCjParams) o;
    return Objects.equals(this.containerTagId, inlineResponse2012OrderDetailsCjParams.containerTagId) &&
        Objects.equals(this.CID, inlineResponse2012OrderDetailsCjParams.CID) &&
        Objects.equals(this.OID, inlineResponse2012OrderDetailsCjParams.OID) &&
        Objects.equals(this.TYPE, inlineResponse2012OrderDetailsCjParams.TYPE) &&
        Objects.equals(this.itEM1, inlineResponse2012OrderDetailsCjParams.itEM1) &&
        Objects.equals(this.amT1, inlineResponse2012OrderDetailsCjParams.amT1) &&
        Objects.equals(this.qtY1, inlineResponse2012OrderDetailsCjParams.qtY1) &&
        Objects.equals(this.CURRENCY, inlineResponse2012OrderDetailsCjParams.CURRENCY);
  }

  @Override
  public int hashCode() {
    return Objects.hash(containerTagId, CID, OID, TYPE, itEM1, amT1, qtY1, CURRENCY);
  }

  @Override
  public String toString() {
    StringBuilder sb = new StringBuilder();
    sb.append("class InlineResponse2012OrderDetailsCjParams {\n");
    
    sb.append("    containerTagId: ").append(toIndentedString(containerTagId)).append("\n");
    sb.append("    CID: ").append(toIndentedString(CID)).append("\n");
    sb.append("    OID: ").append(toIndentedString(OID)).append("\n");
    sb.append("    TYPE: ").append(toIndentedString(TYPE)).append("\n");
    sb.append("    itEM1: ").append(toIndentedString(itEM1)).append("\n");
    sb.append("    amT1: ").append(toIndentedString(amT1)).append("\n");
    sb.append("    qtY1: ").append(toIndentedString(qtY1)).append("\n");
    sb.append("    CURRENCY: ").append(toIndentedString(CURRENCY)).append("\n");
    sb.append("}");
    return sb.toString();
  }

  /**
   * Convert the given object to string with each line indented by 4 spaces
   * (except the first line).
   */
  private String toIndentedString(java.lang.Object o) {
    if (o == null) {
      return "null";
    }
    return o.toString().replace("\n", "\n    ");
  }
}

File: OPENAPI-REST-API/swagger-client/dart/lib/model/inline_response2012_order_details_cj_params.dart
part of swagger.api;

class InlineResponse2012OrderDetailsCjParams {
  
  int containerTagId = null;

  int CID = null;

  String OID = null;

  int TYPE = null;

  String iTEM1 = null;

  int aMT1 = null;

  int qTY1 = null;

  String CURRENCY = null;

  InlineResponse2012OrderDetailsCjParams();

  @override
  String toString() {
    return 'InlineResponse2012OrderDetailsCjParams[containerTagId=$containerTagId, CID=$CID, OID=$OID, TYPE=$TYPE, iTEM1=$iTEM1, aMT1=$aMT1, qTY1=$qTY1, CURRENCY=$CURRENCY, ]';
  }

  InlineResponse2012OrderDetailsCjParams.fromJson(Map<String, dynamic> json) {
    if (json == null) return;
    containerTagId = json['containerTagId'];
    CID = json['CID'];
    OID = json['OID'];
    TYPE = json['TYPE'];
    iTEM1 = json['ITEM1'];
    aMT1 = json['AMT1'];
    qTY1 = json['QTY1'];
    CURRENCY = json['CURRENCY'];
  }

  Map<String, dynamic> toJson() {
    return {
      'containerTagId': containerTagId,
      'CID': CID,
      'OID': OID,
      'TYPE': TYPE,
      'ITEM1': iTEM1,
      'AMT1': aMT1,
      'QTY1': qTY1,
      'CURRENCY': CURRENCY
     };
  }

  static List<InlineResponse2012OrderDetailsCjParams> listFromJson(List<dynamic> json) {
    return json == null ? new List<InlineResponse2012OrderDetailsCjParams>() : json.map((value) => new InlineResponse2012OrderDetailsCjParams.fromJson(value)).toList();
  }

  static Map<String, InlineResponse2012OrderDetailsCjParams> mapFromJson(Map<String, Map<String, dynamic>> json) {
    var map = new Map<String, InlineResponse2012OrderDetailsCjParams>();
    if (json != null && json.length > 0) {
      json.forEach((String key, Map<String, dynamic> value) => map[key] = new InlineResponse2012OrderDetailsCjParams.fromJson(value));
    }
    return map;
  }
}
